.github/workflows/integration.yml

```diff
@@ -49,6 +49,7 @@ on:
         type: choice
         options:
           - minimal
+          - minimal-with-team
```

Comment:
At a later stage it would be great to have both teams and services, and maybe a hello-world deployment. This is obviously fine for now